src/main/SignUp/SignUpScreen.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import { createUser } from './createUserApi'
+
+const SignUpScreen = () => {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const [email, setEmail] = useState(null)
+  const [password, setPassword] = useState(null)
+  const errorMessage = useSelector(state => state.errorMessage)
+
+  const handleOnClick = () => {
+    console.log('email: ', email)
+    console.log('password: ', password)
```

Comment:
I think your settings of the errorMessage should occur here (which might allow you to use local state rather than redux, right?)

Reply:
you mean to clear the error message? If so then yes I fixed that!